asset/cloud-client/src/test/java/com/example/asset/QuickStartIT.java

```diff
@@ -81,4 +81,14 @@ public void testExportAssetExample() throws Exception {
     String got = bout.toString();
     assertThat(got).contains(String.format("uri: \"%s\"", assetDumpPath));
   }
+
+  @Test
+  public void testBatchGetAssetsHistory() throws Exception {
+    String bucketAssetName = String.format("//storage.googleapis.com/%s", bucketName);
+    BatchGetAssetsHistoryExample.main(bucketAssetName);
+    String got = bout.toString();
+    if (!got.isEmpty()) {
```

Comment:
Is there any behavior we want to check if "got" _is_ empty?

Reply:
If "got" is empty, it means the bucket resource change has not been updated into our backend yet. It's rare case but it happens. I don't think there's much we can do for it. Fortunately, it means the API call was successfully, otherwise error response will be returned.